app/graphql/types/date_type.rb

```diff
@@ -2,14 +2,17 @@
 
 module Types
   class DateType < Types::BaseScalar
-    description 'Date type'
+    description 'Date in YYYY-MM-DD format'
+    DATE_FORMAT = '%Y-%m-%d'
```

Comment:
There are no existing mutations that use this type that would be affected by this change, right? (Or if they would be affected, they're already using this format anyway?)

Reply:
Yeah exactly, I just ripped this off of Exchange:

https://github.com/artsy/exchange/blob/master/app/graphql/types/date_type.rb

I'm not sure how this type came to be in the first place, but this way we can parse out dates in a uniform way.